crypto/evp/p_rsa_asn1.c

```diff
@@ -108,6 +108,19 @@ static int rsa_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
   return 1;
 }
 
+static int rsa_pss_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
```

Comment:
Please reference specification and what this function can decode.

Reply:
This method is to call `RSA_parse_public_key` and will have new pss parameters parse added.

The 1st part has RFC refered in its header. The 2nd part will add the RFC when implemented. Existing code only refers RFC again when the spec requires special restrictions (like NULL param for rsaEncryption).
```
// RSA_parse_public_key parses a DER-encoded RSAPublicKey structure (RFC 3447)
// from |cbs| and advances |cbs|. It returns a newly-allocated |RSA| or NULL on
// error.
OPENSSL_EXPORT RSA *RSA_parse_public_key(CBS *cbs);
```